---
title: "MICB425 Portfolio - Microbial Ecological Genomics: From Genomes to Biomes, Spring 2018"
author: "Ryan Lou, 23308142"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output: 
  html_document:
    toc: true
    toc_float: true
    toc_depth: 4
---
# Module 01
    
## Data science Friday  
The remaining second level headers (##) are for separating data science Friday, regular course, and project content. In this module, you will only need to include data science Friday and regular course content; projects will come later in the course.

### Installation check
Third level headers (###) should be used for links to assignments, evidence worksheets, problem sets, and readings, as seen here.  

Use this space to include your installation screenshots.


### Portfolio repo setup
Detail the code you used to create, initialize, and push your portfolio repo to GitHub. This will be helpful as you will need to repeat many of these steps to update your porfolio throughout the course.


### RMarkdown pretty PDF challenge
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


The following assignment is an exercise for the reproduction of this .html document using the RStudio and RMarkdown tools we've shown you in class. Hopefully by the end of this, you won't feel at all the way this poor PhD student does. We're here to help, and when it comes to R, the internet is a really valuable resource. This open-source program has all kinds of tutorials online.


![](http://www.phdcomics.com/comics/archive/phd011718s.gif)


http://phdcomics.com/ Comic posted 1-17-2018


##Challenge Goals
The goal of this R Markdown html challenge is to give you an opportunity to play with a bunch of different RMarkdown formatting. Consider it a chance to flex your RMarkdown muscles. Your goal is to write your own RMarkdown that rebuilds this html document as close to the original as possible. So, yes, this means you get to copy my irreverant tone exactly in your own Markdowns. It's a little window into my psyche. Enjoy =)


**hint: go to the [PhD Comics website](http://phdcomics.com/) to see if you can find the image above**  
*If you can't find that exact image, just find a comparable image from the PhD Comics website and include it in your markdown*



###Here's a header!


Let's be honest, this header is a little arbitrary. But show me that you can reproduce headers with different levels please. This is a level 3 header, for your reference (you can most easily tell this from the table of contents).


####Another header, now with maths

Perhaps you're already really confused by the whole markdown thing. Maybe you're so confused that you've forgotten how to add. Never fear! ~~A calculator~~ R is here:
```{r}
1231521+12341556280987
```


###Table Time


Or maybe, after you've added those numbers, you feel like it's about time for a table!  
I'm going to leave all the guts of the coding here so you can see how libraries (R packages) are loaded into R (more on that later). It's not terribly pretty, but it hints at how R works and how you will use it in the future. The summary function used below is a nice data exploration function that you may use in the ^future^.
```{r}
library(knitr)
kable(summary(cars), caption="I made this table with kable in the knitr package library")  
```


And now you've almost finished your first RMarkdown! Feeling excited? We are! In fact, we're so excited that maybe we need a big finale eh? Here's ours! Include a fun gif of your choice!


![**Such snow. Very fun. Wow.**](http://i.imgur.com/PgksOgG.gif)

###Data Science Assignment 2 - Plotting Data in R

```{r}
library(tidyverse)
library(phyloseq)
metadata <- read.delim("Saanich.metadata.txt")
OTU <- read.delim("Saanich.OTU.txt")
load("phyloseq_object.RData")
```


####Exercise 1
```{r}
ggplot(metadata, aes(x=NO3_uM, y=Depth_m))+
  geom_point(color = "purple", shape=17, size = 2)
```


####Exercise 2
```{r}
#Change from °C to °F
metadata %>%
  mutate(Temperature_F = (Temperature_C*9/5)+32) %>%
  select(Temperature_F, Depth_m) %>%
#Plot data
  ggplot(., aes(x=Temperature_F, y=Depth_m))+
  geom_point()
```


####Exercise 3
```{r}
#convert physeq data to % and plot
physeq_percent = transform_sample_counts(physeq, function(x) 100*x/sum(x))
plot_bar(physeq_percent, fill="Phylum")+
  geom_bar(aes(fill=Phylum), stat = "identity")+
#Add title and axis labels
  ggtitle("Phyla from 10 to 200 m in Saanich Inlet")+
  xlab("Sample depth") + ylab("Percent relative abundance")
```


####Exercise 4
```{r}
#Gather data in 1 column
metadata_nutrients <- metadata %>%
  gather(Nutrient, Concentration, O2_uM, PO4_uM, SiO2_uM, NO3_uM, NH4_uM, NO2_uM)%>%
  select(Depth_m, Nutrient, Concentration)
#Plot facet plot
ggplot(metadata_nutrients, aes(x=Depth_m, y=Concentration))+
  geom_line(color="grey")+
  geom_point()+
  facet_wrap(~Nutrient, scales="free_y")+
  theme(legend.position = "none")+
  ggtitle("Nutrient distribution across a depth of 10-200m in Saanich Inlet")+
  xlab("Depth (m)")+
  ylab("Nutrient Concentration (uM)")
```


###Evidence Worksheet 01


[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)


#### Learning objectives
* Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems. 


#### General questions


#####**What were the main questions being asked?** 

* What is the total number of prokaryote cells in the world?
* How much of the world's carbon is made up by prokaryotes?
* What are the major environments in which these prokaryotes live?
* How does the number if prokaryotes contribute to genetic diversity?
* How does the turnover time vary by environment?

#####**What were the primary methodological approaches used?**

* Performing calculations on data provided in literature cited to provide estimate on prokaryote counts in various representative environments and other habitats
* Extrapolating from equations in literature cited to estimate prokaryote numbers in less accessible environments such as deep sediments
* Creating a depth profile for unconsolidated sediments for subsurface prokaryotes in deep ocean, continential shelf/slope and terrestrial coastal plain using calculated arithmetic averages
* Using assumptions from literature cited regarding the volume and average porosity of the terrestrial subsurface, total pore space occupied by prokaryotes, and average volume of a subsurface prokaryote cell to calculate the total number of terrestrial subsurfadce prokaryotes

#####**Summarize the main results or findings.**

* The habitats that the greatest number of prokayotes are generally categorized as being part of the subsurface (seawater, soil, sediments)
* The total carbon in prokaryotes is estimate to be around 60-100% of the total carbon found in plants
* Terrestrial plants have a lower nutrient (Nitrogen/Phosphorus-containing) content per gram of carbon than soil prokaryotes do
* Mutations are more prevalent in prokaryotes because of their sheer numbers, leading to greater genetic diverty
* Although animals, leaves, and plant tissue have a large resident prokaryote propulation, the scale of those populations are too low to form a major portion of the total number of prokaryotes in the world
    - Eg: Soil has significantly more prokaryotes than leaves do.
    
#####**Do new questions arise from the results?**

* What proportion of the total number of prokaryotes is archaea? 
* What propotion of the total number of prokaryotes is bacteria?
* Is there any way to have a better estimate of the total number of prokaryotes in this day and age better computational tools at our disposal? * How does the prokaryote nutrient pool (C/N/P) differ in the ocean compared to terrestrial environments?

#####**Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?**  


The authors provided a very brief background on what prokaryotes do and how they contribute to genetic diversity, but then quickly goes into describing their methodology as well as the various environments they looked at. However, they explained **how** they know there are lots of prokaryotes in these environments and not **why**, which can be confusing to readers that do not know what prokaryotes are.


The methods were quite vague as they do mention that they extrapolate from the equations of the cited literature, which requires looking at them and determining what they did, especially for unconsolidated sediments. There were assumptions made regarding the porosity of subsurface sediments, pore space taken up by prokaryotes and the volume of prokaryotes, which were used to estimate the total number of prokaryotes in those environments. 


The tables provided were neat and easy to understand and compare the number of prokaryotes in each environment. It also provided a quick way visualizing a calculation that they did to arrive at the total number of prokaryotic cells. Seeing the total number of the cells does not seem far-fetched since the Earth has a high volume in which many small prokaryotic cells can live in.


### Evidence Worksheet 02
#### Learning objectives:
* Comment on the emergence of microbial life and the evolution of Earth systems


[Nisbet *et. al* 2001](https://www.nature.com/articles/35059210)


[Kasting *et.al* 2002](http://science.sciencemag.org/content/296/5570/1066)


[Falkowski *et. al* 2008](http://science.sciencemag.org/content/320/5879/1034)


#### Prompt


##### **Indicate the key events in the evolution of Earth systems at tic marks on the time series.**
* Earth was formed over 4.5 billion years ago
* Life began about 3.8 billion years ago; only hyperthermophiles survived since the ocean was over 100?C in some parts due to the occasional meteorite impact that occured in the past 200 million years
* Cyanobacteria have appeared to evolve 3.5 billion years ago (Rubisco pathway)
* First eukaryotes appeared 2.7 billion years
* Rise of atmospheric O~2~ at 2.2/2.3 billion years ago
    * Most of this is derived from the oceans (cyanobacteria and other phytoplankton), since the bacteria could live under anaerobic conditions and produce O~2~.
* 4.6 billion years ago  
    * Beginning of the Solar System from one or more supernovas nearby
    * Inner planets received carbon and water vapour from the matter in space
    * 4.5 billion years ago
    * Earth was formed around this time 
    * Moon was also formed, giving Earth spin, tilt, day/night cycles and seasons
* 4.4 billion years ago
    * Oldest mineral grains on Earth formed (Zircons): Jack Hills Greenstone Belt
* 4.2-4.1 billion years ago  
    * First evidence of life found using carbon isotopes in graphite in rock/zircon
* 4.0 billion years ago
    * Evidence of plate subduction
* 3.8 billion years ago  
    * Evidence came from the Isua belt in Greenland; oldest known water-lain sedimentary rocks with inorganic carbonate; evidence of oceans 
    * Meteorite impacts largely stopped, allowing for life and stabilizing sea water chemistry and although it was still hot, there were no more impacts that could cause seawater to vaporize
* 3.75 billion years ago  
    * Early methanogenesis by microbes (mostly archaeans)
* 3.5 billion years ago  
    * Fossil microbial biolfilms and stromatolites are evidence of microbes showing up around this time (particularly Archaea), along with carbon fixation/photosynthetic enzymes (Rubisco) in cyanobacteria
* 3.0 billion years ago  
    * First glaciation event - Earth was still brown because of the high amount atmospheric CH~4~ from methanogens and warm despite the dimmer Sun
* 2.7 billion years ago  
    * First eukaryotes emerged
        * They incorporated chloroplast and mitochondria symbionts, which could influence the links between photosynthesis, CO~2~, and N~2~ fixation
    * The great oxidation event largely created by cyanobacteria
    * Terrestrial life started to exist
* 2.2 billion years ago  
    * The microaerobic atmospehre transitioned into oxic air when O~2~ levels sharply increased 
    * Rubisco preferrably takes up the CO~2~ in the atmosphere, leaving O~2~
* 2.1 billion years ago
    * Complex eukaryotes evolved
* 1.3 billion years ago
    * Stromatolites rose sharply in diversity
    * Meiosis and sexual reproduction present in single-celled eukaryotes and also in common ancestor of all eukaroytes, which could have increased genetic diversity and the rate of evolution
* 700 million years ago
    * Snowball Earth; a series of many global glaciations   
* 540 million years ago
    * The Cambrian Explosion - massive increase in multicellular life
* 400 million years ago
    * Devonian explosion 
        * First land plants appeared; trees and seeds evolved
        * A great amount of diversification happened where there were lots of fishes and other species appearing until the Devonian extinction where most of those species disappeared
    * Gigantism (large animals and plants) occured
* 252 million years ago
    * Another mass extinction event around this time near the end of the Permian Period "the Great Dying"
    * Gave rise to the "Age of the dinosaurs" where they dominated the Earth for about 186 million years
* 20 million years ago
    * Dramatic global warming occured
    * Mammals and other multicellular organisms diversified further and increased in size
* 550,000 years ago
    * Beginning of the series of modern "Ice Ages"
* 200,000 years ago
    * Emergence of the first human being
    
#### **Describe the dominant physical and chemical characteristics of Earth systems at waypoints.**
* Hadean  
    * Meteorite impacts throughout this eon 
    * More CO~2~ as evidenced by limestone and much higher vapour pressure due to higher temperatures (500?C)
    * Temperature started to cool down even more closer to the end of the eon (100?C)
    * Surface ocean temperatures were about 100°C
    * Silica-containing rocks provided evidence for waters in the ocean because the lithosphere would need to be hydrated and cooled for these rocks to form.
* Archean  
    * Methanogenesis started early on in the eon and the atmosphere was predominately made of CH~4~, with some CO~2~ due to the lack of oxygen
        * The large amount of CH~4~ is necessary for the Earth as a greenhouse gas because keeps the heat on Earth especially when the sun was 33% dimmer; was produced in levels 10-100x higher before O~2~ became common in the atmosphere
            * Having too high of a CH~4~:CO~2~ ratio would create a smog too dense that it could block UV light
        * CH~4~ could be used to build biomass
        * As Rubisco preferrentially takes up CO~2~, this leaves even more CH~4~ relative to CO~2~ in the atmosphere at the time when it became more common
    * Also sulfur reduction occured when there was no oxygen
    * Atmosphere is starting to become more oxic with time
* Proterozoic  
    * Big increase in O~2~ concentrations in the atmopshere
    * Plenty of CO~2~ in atmosphere compared to CH~4~
* Phanerozoic  
    * Relatively warm temperatures dring the Devonian
    * Pangaea formed and there was dry, harsh climate in the interior of the supercontinent
    * After Pangaea split, the Atlantic Ocean formed near the extinction of dinosaurs
    * Climate heated up during the Neogene


### Evidence Worksheet 03 


[Waters *et al* 2016](http://science.sciencemag.org/content/351/6269/aad2622)

#### Learning objectives
* Evaluate human impacts on the ecology and biogeochemistry of Earth systems. 

#### General questions


#####**What were the main questions being asked?** 

1.	What are the signals that represent different times during the Holocene?
2.	How does the rapid increase in anthropogenic materials act as a signature for a certain time periods?
3.	What geochemical signatures have humans created in sediments and ice?
4.	How have the testing and deployment of nuclear weapons created a traceable signature in sediments and ice?
5.	How has atmospheric CO2 levels changed according to the ice cores and sea levels?


#####**What were the primary methodological approaches used?**


This paper is a review that explained various geochemical signatures and trends during the Holocene with other literature, and the Anthropocene. It appears from the original paper that there were several different measures that were analyzed. Some measures include taking lake sediment cores to measure for constituents from anthropogenic materials of that sediment, glacier ice cores to measure atmospheric CO2, measuring sea level increases. There are also other key anthropogenic change markers such as temperature anomaly, methane as well as human made materials like concrete, plastic and black carbon from fossil fuel emissions.


#####**Summarize the main results or findings.**

As technological advances occur through time, human population growth increases, leading to a corresponding increase in resource usage (fossil fuels, metals, fertilizers, land use change), leading to an ecological change. As show in the original paper, there is a considerable increase in ice core nitrate levels, atmospheric CO2, methane, temperature, cement, plastic, black carbon, which can all be attributed to humans. 


Human wastes and byproducts of production has caused the formation of persistent materials in the world, which can act as a geochemical signature. Plastics produced by humans have started accumulating in both shallow and deep water sediments which can form biological and/or chemical interactions.


Radioactive materials have also started appearing due to the testing and deployment of nuclear arms by the USA and Russia. This created an increase on 239Pu (Plutonium) from the fall out and can accumulate in the sediments, creating another signature of the Anthropocene given its long half life.


There has been a relatively large proportion of animals going extinct, due to intensive human land and resource usage.
    
    
#####**Do new questions arise from the results?**


1.	How are the time divisions of the Anthropocene going to be defined?
2.	Can we even determine when the Anthropocene started?
3.	How has the increase in anthropogenic materials affected the geochemical processes on Earth?


#####**Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?**  


Overall, reading the paper wasn't too much of a challenge and since it is partially on a topic that is well known (global warming and human activities), it made it easier to understand the trends in the figures. The author explained the trends in the figures and what it relates to in terms of human activities. However, since it is a review, the methods were explained, but not in as much detail as the primary paper. Also, the ratios (such as 18O/16O), were not well explained so I was unable to see how it would relate to what it meant in the context of the Anthropocene. The authors do provide a good foundation with questions that would drive further research into defining the Anthropocene.


### Problem set 01
#### Learning objectives: 
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific questions:
#####**What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text.**


The primrary prokaryotic habitats on Earth are **aquatic environments** (both fresh water and marine) across various regions in the open ocean and sediments, which has a total prokaryotic cell abundance of 1.2 x 10^29^, **soil**, which has a total prokaryotic cell abundance of 2.6 x 10^29^, and **subsurface**, which has a total prokaryotic cell abundance of 4.9 x 10^30^.


#####**What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including Prochlorococcus? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth?** 


The estimated cell abundance of prokaryotic cells in the upper 200m of the ocean is 3.6 x 10^28^ and marine autotrophs make up 8% of those prokaryotic cells. This is significant as the autotrophs control the majority of the primary production in the ocean and provides oxygen to the atmosphere and carbon to marine heterotrophic bacteria.


#####**What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?**


Autotrophs are "self-nourishing" organisms that can fix inorganic carbon into biomass  
Heterotrophs are organisms that can assimilate orgnaic carbon  
Lithotrophs are organisms that can use inorganic substrates 


#####**Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth? ** 


The deepest habitat capable of supporting prokaryotic life is in the terrestrial subsurface where it can reach depths up to 4 km, as mentioned in the text. For marine environments, the deepest environment is the Marianas Trench, which is around 10.9 km deep, and possibly 5-6 km deeper before the temperature reaches the limiting factor of having an average temperature of 125 degrees celsius. 


#####**Based on information provided in the text your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height?**  


The highest habitat capable of supporting prokaryotic life is at 57-77 km in the atmosphere, but the abundance is relatively low because the oxygne concentration at that height is relatively low. There could also be a low amount of water vapour, which maybe needed by those prokaryotes.


#####**Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth's biosphere measured in km?**


The vertical distance of the Earth's biosphere is measured to be around 57-77 km in the atmisohere due to prokaryotes being detected at those heights.


#####**How was annual cellular production of prokaryotes described in Table 7 column four determined? (*Provide an example of the calculation*)**


It was determined by dividing the number of days in the year (365) by the turnover time and then multiplying it by the population size.


Example (for Marine Heterotrophs living above 200m  ; Turnover time = 16 days; Population size = 3.6 x 10^28^):

$$\frac{365 \ days/year}{16 \ days} \cdot{3.6\times 10^{28}} \ Cells = 8.2\times 10^{29} \ Cells/year$$ 


A large number of these prokaryotes are lysed by marine viruses.


#####**What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?** 


Carbon efficiency = 20%  
5-20 fg C/cell  
$$3.6\times 10^{28} \ cells \cdot{6\times 10^{-30}} \ Pg \ C /cell = 0.72 \ Pg \ C$$
Pg C in Marine Environments


$$0.72 \ Pg \ C\times{4} = 2.88 \ Pg \ C$$


#####**How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)**


To account for the chance of having four simultaneous mutations in shared genes for marine heterotrophs above a depth of 200m:
$$(4\times 10^{-7})^{4} \ mutations/gene/DNA \ replication = 2.56\times10^{-26} \ $$


To
$$3.6\times 10^{28} \ Cells \cdot 22.8 \ year^{-1} = 8.2\times 10^{29} \ Cells/year$$
$$8.2\times 10^{29} \cdot 2.56\times 10^{-26} = 2.1\times 10^{4} \ {mutations/year}$$

#####**Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?**


* Having a large population size, increases the mutation rate significantly, thereby increasing genetic diversity since more mutations can give rise to new traits, which can be helpful for adapting to a new environment. This increases adaptive potential. Although point mutations can be largely silent, they have the ability to diversify microbial genomes, and they are not the only mode since there is also horizontal gene transfer, which can happen with microbes.


#####**What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text?**




### Problem set 02
#### Specific Questions:
##### **What are the primary geophysical and biogeochemical processes that create and sustain conditions for life on Earth? How do abiotic versus biotic processes vary with respect to matter and energy transformation and how are they interconnected?**


Tectonic movement along with photochemical reactions in the atmosphere provide a flow of substrates to organisms and can remove metabolic products through large scale processes, creating geochemical cycles.This allows elements and molecules to interact, forming bonds that can be formed and broken in a cyclical fashion. Abiotic processes generally include the acid/base chemistry (proton transfer), the geophysical processes that bring about chemical reactions. Biotic processes are brought about by the photochemical reactions, which drive primary production and also the cycling of nutrients such as N and P. This primary production can support heterotrophic bacteria that can respire organic matter and remineralize inorganic substrates that contain N, S, and P. These biotic processes also contribute to oxygen production and depletion.


Biogeochemical: 
 (Redox)
 
 
##### **Why is the Earth's redox state considered an emergent property?**


The Earth's redox state is considered an emergent property as it is reliant on reactions that occur in the Earth's (bio)geochemical processes that occur due to regular chemical redox reactions and the metabolic processes of microbes. No single organism in these processes possess all the components that is needed to run these systems, and it is instead the reactions that occur, which are connected to each other that make up the processes as a whole.


##### **How do reversible electron transfer reactions give rise to element and nutrient cycles at different ecological scales? What strategies do microbes use to overcome thermodynamic barriers to reversible electron flow?**


Reversible electron transfer reactions give rise to new element and nutrient cycles as these cycles would have been almost impossible given the thermodynamic circumstances that can be corrected by this process. More so, the thermodynamic conditions in the environment can influence the rate of each  reaction, determining whether it is spontaneous or not (ex: abundance of substrates, products, etc.).


##### **Using information provided in the text, describe how the nitrogen cycle partitions between different redox, niches, and microbial groups. Is there a relationship between the nitrogen cycle and climate change?**


The different stages of the nitrogen cycle all require different amounts of oxygen , corresponding to different redox niches, and the other microbes that will be in that environment. In nitrogen fixation, nitrogen gas is fixed from our atmosphere into ammonium through the use of nitrogenase, which must be protected from oxygen. Another source of fixed nitrogen is through the oxidation of ammonium to nitrite and finally nitrate in aerobic environments and provides nitrates to phytoplankton. On the other hand, denitrification, the anaerobic oxidation of nitrate and nitrite to nitrogen gas occurs in oxygen minimum zones in the ocean.


The nitrogen cycle is indirectly connected to global warming. All microbes require nitrogen to synthesize protein and nucleic acids as they contain amino acids and nitrogenous bases respectively. A major mode of N~2~ fixation is through microbes, although the Haber-Bosch process contributes to a significant portion as well. The nitrogen cycle is what controls the amount of available fixed nitrogen to organisms, which can influence the abundance of microbes how how they affect the atmosphere, especially through photosynthesis. A shift in the nitrogen cycle could have big implications on not just the microbes, but on the community itself and the overall climate, albeit not directly since it only deals with nitrogen and hydrogen gas, which are emitted through nitrogen cycle processes. 


##### **What is the relationship between microbial diversity and metabolic diversity and how does this relate to the discovery of new protein families from microbial community genomes?**


Although genes may mutate fairly often, there will always be a set of core genes that remain and are protected since they are extremely important. These genes code for the major redox reactions essential for life and biogeochemical cycles. Therefore, microbial diversity with new genes that pop up does not necessarily mean there is diversity in metabolic protein since they can be crowded out by those genes that have been conserved.


Given that there are almost limitless ways to mutate genes, this leads to the hypothesis of having limitless evolutionary diversity in nature. This coupled with the fact that the rate of discovery of unique protein families has been proportional to the sampling effort, means that our sampling effort is the limiting factor and the rate at which we discover new protein families scales approximately linearly with the number of new genomes sequenced.


##### **On what basis do the authors consider microbes the guardians of metabolism?**


Temporary and simultaneous guardianship because of the nature of microbial evolution from horizontal and vertical gene transfer, which can change which phenotype is dominant at the time. A dominant phenotype protects the metabolic pathway in the environment, and if it does not survive environmental perturbations applying selective pressures on pathway genes, then it will disappear. Humans could possibly replicate the individual pathways, but the overall metabolic biogeochemical processes that control the flow of electrons can only be done by microbes.


###Writing Assessment_01


Microbes have existed for many eons and with them, comes their innate ability to impact biogeochemical cycles, such as the nitrogen cycle, through the numerous metabolic pathways that exist on terrestrial and marine environments that can affect primary productivity in both a negative and positive fashion (1).  Although microbes are unseen through the world given their microscopic size, they contribute to a significant of the world's biomass due to their abundance in a wide variety of environments and their sheer abundance contributes to a significant portion of the global catalysis (2). Given that microbes drive many large-scale processes across the globe, it could be said that form some sort of "engine" that drives the Earth's biogeochemical cycles and can therefore be affected by, and affect climate change (3). As humans have existed for a relatively short period of time in the history of the Earth (~300,000 years ago) (4) compared to microbes, it could be said that humans heavily rely on microbes, whereby the opposite cannot be said in the same light. 


The nitrogen cycle is an essential part of the Earth's processes, especially affecting that affect the biological landscape through processes put forth by microbes. Nitrogen fixation reduces the abundant N2 gas in the atmosphere into ammonium, a form that can be used by organisms for assimilation into biomass (1). This ammonium can be further oxidized by microbes to nitrite and finally nitrate, which can be used by phytoplankton to drive primary production, producing oxygen and biomass that can support marine food webs (1). Also, heterotrophic bacteria can remineralize This oxygenation of our atmosphere is essential for supporting the lives of us humans and since they partially contribute to, and also support the organisms that contribute 50% of the world's primary productivity (2). However, as this is a cycle, there are also processes that go in the opposite direction where fixed nitrogen is lost as nitrogen gas (N2) or nitrous oxide (N2O), whether it be through denitrification or annamox performed by microbes (Canfield, 2010). As these processes usually happen in the absence of oxygen, they happen in zones in the ocean called Oxygen Minimum Zones or OMZs (5). This causes the nitrogen to be released into atmosphere again and nitrous oxide can be a greenhouse gas, potentially causing problems with climate change, which increases the CO?2 in the atmosphere and temperature, reducing the solubility of oxygen causing the increase of these OMZs and further exacerbating climate change in the process (6). 


![Figure 1: the reactions and linkages of the nitrogen cycle and the metabolic genes involved (1).](https://i.imgur.com/uA8n42Z.png)


Microbial cells are microscopic in size and only contain a small amount of carbon in each cell, but they are very abundant around the Earth and the high number of cells contributes to turning these biogeochemical processes into large-scale ones. There are a wide variety of environments such as soil and the oceans that vary in microbial abundance largely due to differing conditions. Marine environments typically have around 1.5 x 1029 prokaryotic cells and typically contribute a little over 50% of the world's primary productivity at 51 Pg of carbon per year, largely due to subsurface prokaryotes (2). Soil has almost double the number of cells at 2.6 x 1029, and they contribute to the break down of organic matter and recycling of nutrients for other organisms like plants to grow (2). Although the subsurface (both marine and subsurface) is hard to measure, it could be said that there is an order of magnitude greater than both marine and soil prokaryotes combined and these are generally anaerobic bacteria that live in the sediments under the sea or terrestrial surface and reduce inorganic to methane (CH4) or sulfate to hydrogen sulfide (H2S) (2). These pathways have evolved from several billion years ago when methanogenesis was common (7) and have been conserved over the years showing that microbes have been around for many years. With this many prokaryotic cells, it could be said that they contribute to a large portion of the world's biomass and global biogeochemical catalysis.


Given that microbes have been on Earth at a time not too relatively long after it was formed (about 3.8 billion years ago) (7), this meant that microbes have been on t he Earth significantly longer than humans have. This meant that the metabolic pathways that microbes partake in that we see today have survived over these years have withstood the evolutionary pressures that arise along the way and have been refined through small perturbations, making them "guardians of metabolism" through the vertical and horizontal gene transfer (3). As humans have only been around for around ~300,000 years (4), a relatively very small amount of time on the geologic time scale, we have largely depended on microbes to previously shape the world we have lived him for so many years, including all the biogeochemical cycles and landscape that we take for granted through the control of electron flow (3). Although microbes do play a role in climate change, it largely depends on human activities that release large amounts of CO2 into the atmosphere and the cycling of microbes could mitigate it, but it takes time. This makes us dependent on microbes and microbes could continue living without us and could change the Earth even more over longer time scales.


Overall, microbes have a large impact on the biogeochemical cycles of the Earth due to their sheer abundances in the various environments that make up the Earth. They influence the nutrient availability for other organisms, including humans through their processes, like nitrogen cycling and the pathways that come with that, and photosynthesis with primary production producing oxygen for humans and animals. As we came not too long ago, but several billions and millions of years after microbes started appearing, they could easily live without us given that their metabolic pathways have survived over time, while we cannot live with them since we rely on their processes to shape the Earth like it is now. However, given our rate of technological advances, is there some way we could sharply reduce our reliance on microbes?

References:


1.	Canfield DE, Glazer AN, Falkowski PG. 2010. The Evolution and Future of Earth's Nitrogen Cycle. Science. 330(6001):192-196. 10.1126/science.1186120


2.	Whitman WB, Coleman DC, Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci USA. 95(12):6578-6583.


3.	Falkowski PG, Fenchel T, Delong EF. 2008. The Microbial Engines That Drive Earth's Biogeochemical Cycles. Science. 320(5879):1034-1039. 10.1126/science.1153213


4.	Schlebusch CM, Malmstrom H, Gunter T, Sjodin P, Coutinho A, Edlund H, Munters AK, Vicente M, Steyn M, Sooyall H, Lombard M, Jakobsson, M. 2017. Southern African ancient genomes estimate modern human divergence to 350,000 to 260,000 years ago. Science. 358(6363):652. 10.1126/science.aao6266


5.	Lam P, Kuypers MMM. 2011. Microbial Nitrogen Cycling Processes in Oxygen Minimum Zones. Ann Rev Mar Sci. 3(1):317-345. 10.1146/annurev-marine-120709-142814


6.	Stramma L, Visbeck M, Brandt P, Tanhua T, Wallace D. 2009. Deoxygenation in the oxygen minimum zone of the eastern tropical North Atlantic. Geophys Res Lett. 36(20):L20607. 10.1029/2009GL039593


7.	Nisbet EG, Sleep NH. 2001. The habitat and nature of early life. Nature. 409(6823):1083-1091. 10.1038/35059210


###Module_01 References


Whitman WB, Coleman DC, Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci USA. 95(12):6578-6583.  
[PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)


Kasting JF, Siefert JL. 2002. Life and the Evolution of Earth's Atmosphere. Science. 296(5570):1066-1068.
[10.1126/science.1071184](http://science.sciencemag.org/content/296/5570/1066)


Nisbet EG, Sleep NH. 2001. The habitat and nature of early life. Nature. 409(6823):1083-1091.
[10.1038/35059210](https://www.nature.com/articles/35059210)


Falkowski PG, Fenchel T, Delong EF. 2008. The Microbial Engines That Drive Earth's Biogeochemical Cycles. Science. 320(5879):1034-1039.
[10.1126/science.1153213](http://science.sciencemag.org/content/320/5879/1034)


Waters CN, Zalasiewicz J, Summerhayes C, Barnosky AD, Poirier C. Gluszka, Cearreta A, Edgeworth M, Ellis EC, Ellis M, Jeandel C, Leinfelder R, McNeill JR, Richter DD, Steffen W, Syvitski J, Vidas D, Wagreich M, Williams M, An Z, Grinevald J, Odada E, Oreskes N, Wolfe AP. 2016. The Anthropocene is functionally and stratigraphically distinct from the Holocene. Science. 351(6269):aad2622-1-2622-10. 
[10.1126/science.aad2622](http://science.sciencemag.org/content/351/6269/aad2622)


#Module 02
### Evidence Worksheet 04 


[Martinez *et al* 2007](http://www.pnas.org/content/104/13/5590)


#### Learning objectives


* Discuss the relationship between microbial community structure and metabolic diversity
* Evaluate common methods for studying the diversity of microbial communities
* Recognize basic design elements in metagenomic workflows


#### General questions


#####**What were the main questions being asked?** 


1.	What is the minimum number of genes required to express Proteorhodopsin?
2.	Other than providing a light-powered proton pump, how does having Proteorhopdsin confer a fitness advantage?
3.	How can Proteorhodopsin be introduced in a heterologous host?
4.	How do photopigments vary in Proteorhodopsin-containing photophosphorylating microbes? 


#####**What were the primary methodological approaches used?**


* Planktonic DNA cloned into a fosmid vector, which is transformed into E. coli, cultured and colonies were screened for orange/red colour (the pigment of interest).
* These pigments (carotenoids) were extracted and analyzed, protecting it from photooxidation, to detect for photopigment expression
* Measure proton-pumping of the Proteorhodopsin using pH as a proxy
* Measure ATP production by measuring ATP production from respiration in the dark as a baseline and then measure the ATP produced in the presence of light
* Knocking out Proteorhodopsin genes to determine their function


#####**Summarize the main results or findings.**


Proteorhodopsin needs retinal to function, but does not need an exogenous pigment to be added when added to a heterologous host since synthesizing retinal requires its precursor, ??-carotene and that produces the orange/red colour.

Proteorhodopsin can be transferred vertically and can be well conserved, but can easily be transferred through horizontal gene transfer since there only being 7 genes needing to be transferred. These genes are required to catalyze the biosynthesis of retinal from β-carotene.

There is a fitness benefit from this since they can pump protons, creating a proton gradient to produce more ATP in the presence of light.

    
#####**Do new questions arise from the results?**


1.	Can Proteorhodopsin be transferred to a wider range of microbes than what has been discovered so far?
2.	Does Proteorhodopsin have any negative effects on the fitness of a microbe?
3.	What other Proteorhodopsin gene clones are there?
4.	Do ecological and/or physiological conditions create selective pressures for Proteorhodopsin?


#####**Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?**  


The authors do provide a quick background on what Proteorhodopsins do and present how they can contribute to a fitness advantage in a logical way, but they do not explain the mechanism behind it fully. I was able to understand the paper, but not fully. Certain parts, like the methods and results, are filled with jargon and that made it hard to understand as certain words, like trans-retinal were not defined. However, this made the methods detailed, but harder to follow. The diagrams were easy to understand and help with understanding the paper.


### Problem set_03 “Metagenomics: Genomic Analysis of Microbial Communities”
#### Learning objectives: 
Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.  


#### Specific Questions:
* How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?
  

2016:
* 89 Bacterial Phyla
* 20 Archaeal Phyla
    * Phyla was found using the 16S rRNA databases
* **Could be up to 1500 due to microbes that live in the "Shadow Biospere"**


2003:
* 26 of 52 major bacterial phyla have been cultured 


* How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?
    * There are about 110,217 on the EBI database (can be lower due to some repos being private/still being worked on )
    * Environment types (sediment, gut, soil, aquatic), which are environments that are hard to culture from
    
    
* What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)?
    * **Shotgun Metagenomics:**
        * Assembly:
            * EULER
            * Velvet
        * Binning - S-GCOM
        * Annotation - KEGG
        * Analysis Pipelines - Megan 5
        * Databases:
            * LMG/M, MG-RAST, NLB 1
    * **Marker Gene Metagenomics:**
        * Standalone Software: OTUbase
        * Analysis pipelines: 
            * SILVA
        * Denoising:
            * Amplicon Noise
            * UCHIME
        * Databases:
            * Ribosomal Database Project (RDP)
            * Greengenes
            * SILVA
    * **General Use**
        * Alignment - BWA (aln and mem)
        * BLAST
        
    
* What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?   
    * Phlogenetic
* What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?  


Metagenomic sequence binning is the proces from grouping sequences that come from a single genome.
    * **Algorithm types**
        * Align genome to database (BWA)
        * Group to bins based on DNA characteristics based on DNA characteristics: GC Content, Codon usage
    * ** Risks and opportunities in binning**
      
    
* Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?  
    * Functional screens (biochemical, etc.)
    * 3rd Generation sequencing (Nanopore)
    * Single cell sequencing
    * FISH probe
    
    
###Module_02 references


Martinez A, Bradley AS, Waldbauer JR, Summons RE, DeLong EF. 2007. Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host. Proc Nat Acad Sci U S A. 104(13):5590-5595.
[10.1073/pnas.0611470104](http://www.pnas.org/content/104/13/5590)


Wooley JC, Godzik A, Friedberg I. 2009. A Primer on Metagenomics. PLoS Comput Biol. 6(2):e1000667.
[10.1371/journal.pcbi.1000667](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000667)

#Module 03
###Problem_Set_04
```{r}
library(phyloseq)
library(knitr)
library(kableExtra)
library(tidyverse)
```


####Part 1
```{r}
#Enter in subsample data
subsample_bag3 = data.frame(
  Species_number = c(seq(1,32, by=1)),
  Species_type = c("MMs","MMs","MMs","MMs","MMs","MMs",
                   "Skittles","Skittles","Skittles","Skittles",
                  "Skittles","Gummy_Bears","Gummy_Bears","Gummy_Bears",
                   "Gummy_Bears","Gummy_Bears","Gummy_Bears","Rods","Rods",
                   "Rods","Rods","Rods","Circular","Filamentous",
                   "Filamentous","Filamentous", "Brick","Brick",
                   "Round","Round","Round","Round"),
  Species_name = c("Orange", "Yellow","Green","Red", "Brown","Blue",
                   "Yellow","Orange","Red","Brown","Green",
                   "Orange", "Pink","Yellow","White","Red","Green",
                   "Yellow","Orange","Pink","Green","Red",
                   "Blue_Swirl", "String_like","Spider",
                   "Coke_Bottle","Big_blue","Small_blue","Green", "Yellow",
                   "Red","Purple"),
  Abundance = c(13,8,4,3,4,12,6,7,4,10,8,1,3,2,4,3,4,5,8,5,2,4,
                1,1,1,1,1,1,1,1,1,1)
)
```

```{r}
subsample_bag3 %>%
  kable("html") %>%
  kable_styling(bootstrap_options = "striped", font_size = 10,
                full_width = F)
```


####Part 2
```{r}
collectors_curve = data.frame(
  x = c(seq(1, 130,by = 1)),
  y = c(1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,
        3,3,3,3,4,4,4,5,5,5,5,6,6,6,6,6,6,6,6,6,6,
        6,6,7,7,7,7,7,7,8,8,8,8,8,8,8,9,9,9,9,10,10,10,
        10,10,10,10,10,10,10,11,11,11,11,11,11,11,11,12,
        13,13,13,14,14,15,15,15,15,16,16,16,17,17,17,17,
        18,18,18,18,18,19,19,19,19,19,19,19,19,20,20,20,20,
        20,21,21,22,22,22,22,23,24,25,26,27,28,29,30,31,32)
)

ggplot(collectors_curve, aes(x = x, y = y)) +
         geom_point() +
         geom_smooth()+
  labs(x="Cumulative number of individuals classified", 
       y="Cumulative number of species observed", title = "Collector's Curve of a subsample of Bag #3")
```


####Part 3
```{r}
#Invsimpson for M&M species
1/((13/44)^2+(8/44)^2+(4/44)^2+(3/44)^2+(4/44)^2+(12/44)^2)

#Invsimpson for Skittle species 
1/((6/35)^2+(7/35)^2+(4/35)^2+(10/35)^2+(8/35)^2)

#Invsimpson for Gummy Bears species
1/((1/17)^2+(3/17)^2+(2/17)^2+(4/17)^2+(3/17)^2+(4/17)^2)

#Invsimpson for Rod species 
1/((5/24)^2+(8/24)^2+(5/24)^2+(2/24)^2+(4/24)^2)

#Invsimpson for Circular species
1/((1/1)^2)

#Invsimpson for Filamentous species
1/((1/3)^2+(1/3)^2+(1/3)^2)

#Invsimpson for Brick species
1/((1/2)^2+(1/2)^2)

#Invsimpson for Round species 
1/((1/4)^2+(1/4)^2+(1/4)^2+(1/4)^2)
```
```{r}
#Calculate inverse simpson for whole sample
1/((13/130)^2+(8/130)^2+(4/130)^2+(3/130)^2+
                            (4/130)^2+(12/130)^2+(6/130)^2+(7/130)^2+
                            (4/130)^2+(10/130)^2+(8/130)^2+(1/130)^2+
                            (3/130)^2+(2/130)^2+(4/130)^2+(3/130)^2+(4/130)^2+
                            (5/130)^2+(8/130)^2+(5/130)^2+(2/130)^2+(4/130)^2+
                            (1/130)^2+(1/130)^2+(1/130)^2+(1/130)^2+(1/130)^2+
                        (1/130)^2+(1/130)^2+(1/130)^2+(1/130)^2)

## Calculate Chao1 - Singletons = 11; Species observed >2 times= 21 
32 + ((11^2)/(2*21))
```


####Part 4
```{r}
#Calculate diversity using Vegan
library(vegan)

subsample_bag3_diversity = 
  subsample_bag3 %>%
  select(Species_number, Abundance) %>%
  spread(Species_number, Abundance)
#Subsample inverse simpson using Vegan
diversity(subsample_bag3_diversity, index ="invsimpson")
#Subsample chao1 using Vegan
specpool(subsample_bag3_diversity)
```
These calculations are different from the previous calculations by 0.02 for inverse simpson and 2.88 for chao1.


####Part 5
1. The measure of diversity depends on the definition of species in the sample since, especially chao1 since the number of species can influence the observed number of species, along with the number of singletons in the sample. 


2. Removing the colour separation would significantly decrease the number of species, combining the skittles and M&Ms would also decrease the number of species.


3. Different sequencing technologies can introduce biases that can overrepresent one species relative to another. Also, there could be varying error rates in different sequencing technologies, which can affect the categorization of species, affecting the number of species.


### Evidence Worksheet 05


[Welch *et al* 2002](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC139262/)


#### Learning objectives


* Evaluate the concept of microbial species based on environmental surveys and cultivation studies.
* Explain the relationship between microdiversity, genomic diversity and metabolic potential
* Comment on the forces mediating divergence and cohesion in natural microbial communities


#### General questions


#####**What were the main questions being asked?** 


1.	What are the differences between these three strains, beyond their 16S rRNA sequence?
2.	Other than providing a light-powered proton pump, how does having Proteorhopdsin confer a fitness advantage?
3.	How can Proteorhodopsin be introduced in a heterologous host?
4.	How do photopigments vary in Proteorhodopsin-containing photophosphorylating microbes? 


#####**What were the primary methodological approaches used?**


* Sanger sequencing was used to sequence the genomes (dye terminator chemistry) with 7X coverage
* Genome annotation using a web-based annotator called MAGPIE
    * Uses GLIMMER to look for and define ORFs
* Use BLAST to check predicted proteins for redundance


#####**Summarize the main results or findings.**


* Backbone was acquired through vertical gene transfer, while pathogenicity islands can be acquired through horizontal gene transfer
* Pathogenicity islands were acquired through horizontal gene transfer causes large variations between different strains 
    * Absence of Type III secretion system in CFT073
    * Absence of phage/plasmid-encoded virulence genes common to strain O157:H7
* All three strains encode a pili similar to Salmonella Enterica, but vary in their genetic makeup that encode these fimbriae or pili
* CFT073 differs from MG1655 and EDL933 in its protein amino acid sequence by 53 to 81%

    
#####**Do new questions arise from the results?**


* Does this large variance in pathogenicity islands apply to other bacteria that have pathogenic and non-pathogenic strains? 
* Is using the 16S rRNA sequence sufficient for determining difference species given that there can be high variability between strains?


#####**Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?**  


* The background information is brief and does not give enough background for the reader to understand what is being looked at 
* The methods section is very brief (2 paragraphs) and does not fully explain the techniques that they used
* Better explanation of the secretion systems would help in understanding how it is linked fimbriae/pili formation


###Module_03 references


Welch RA, Burland V, Plunket III G, Redford P, Roesch P, Rasko D, Buckles EL, Liou SR, Boutin A, Hackett J, Stroud D, Mayhew GF, Rose DJ, Zhou S, Schwartz DC, Perna NT, Mobley HLT, Donnenberg MS, Blattner FR. 2002. Extensive mosaic structure revealed by the complete genome sequence of uropathogenic *Escherichia coli*. Proc Nat Acad Sci U S A. 99(26):17020-17024.
[PMC139262](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC139262/)
